default.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":combinePatchMinorReleases",
+    ":dependencyDashboard",
+    ":pinAllExceptPeerDependencies",
+    ":separateMajorReleases",
+    "config:base",
+    "group:monorepos",
+    "schedule:weekends",
+    "github>bitwarden/renovate-config:pin-actions"
```

Comment:
⛏️ Sort these.

Reply:
I kind of wanted the external ones last. Could probably add a newline between them to make it explicit?

Reply:
I saw https://github.com/bitwarden/server/blob/main/.github/renovate.json#L5 is all. I am not picky.